Taskfile.yml

```diff
@@ -125,8 +125,8 @@ tasks:
     cmds:
       - docker compose {{ .DOCKER_COMPOSE_FILES }} pull
 
-  dev:reset:
-    desc: Create local development setup in a clean state
+  dev:reset:common:
```

Comment:
What is the purpose of dev:reset:common? Would be nice to have in the description what the intention is.
Also consider which of the newly introduced dev:rest* cmds private so it is clear what is meant to be run as seprate task cmds and what is used internally